components/raftstore/src/store/config.rs

```diff
@@ -193,7 +195,7 @@ impl Default for Config {
             raft_reject_transfer_leader_duration: ReadableDuration::secs(3),
             split_region_check_tick_interval: ReadableDuration::secs(10),
             region_split_check_diff: split_size / 16,
-            clean_stale_peer_delay: ReadableDuration::minutes(10),
+            clean_stale_peer_delay: ReadableDuration::minutes(11),
```

Comment:
Why change it?

Reply:
Because the following code in `TiKvConfig:: compatible_adjust` had deleted, previous `compatible_adjust` just run once when tikv setup, but with `enable-dynamic-config`, `compatible_adjust` will run more then once for a config, and every time the following code will change `clean_stale_peer_delay` no matter the value has change or not. So rather use the following code we can just change the default value of `clean_stale_peer_delay`, since both `clean_stale_peer_delay` and `end_point_request_max_handle_duration` are not support online changing.
```
if self.raft_store.clean_stale_peer_delay.as_secs() > 0 {
        let delay_secs = self.raft_store.clean_stale_peer_delay.as_secs() + 
self.server.end_point_request_max_handle_duration.as_secs();
        self.raft_store.clean_stale_peer_delay = ReadableDuration::secs(delay_secs);
}
```